test/plugin/test_parser_nginx.rb

```diff
@@ -24,23 +25,23 @@ def create_driver
 
   def test_parse
     d = create_driver
-    d.instance.parse('127.0.0.1 192.168.0.1 - [28/Feb/2013:12:00:00 +0900] "GET / HTTP/1.1" 200 777 "-" "Opera/12.0"') { |time, record|
+    d.instance.parse('127.0.0.1 192.168.0.1 - [28/Feb/2013:12:00:00 +0900] "GET / HTTP/1.1" 200 777 "-" "Opera/12.0" -') { |time, record|
```

Comment:
Don't change existing test because it is needed to check compatibility.

Reply:
I see, this test will fail though if it won't be adapted to the nginx log format, how are you suggesting to overcome this issue? Create another test with the new format?

Reply:
> Create another test

Yes, creating new test is needed. Many users use nginx format so we can't break it. It breaks user environment and causes log lost.

> the new format?

One regular expression seems to support log with/without http_x_forwarded_for.